kube/file-vault/file-vault-ingress.yml

```diff
@@ -9,9 +9,7 @@ metadata:
   {{ end }}
 {{ file .FILEVAULT_INGRESS_EXTERNAL_ANNOTATIONS | indent 2 }}
 spec:
-  {{ if or (eq .KUBE_NAMESPACE .BRANCH_ENV) (eq .KUBE_NAMESPACE .UAT_ENV) }}
   ingressClassName: nginx-external
```

Comment:
@adityababumallisettiHO Just a note - also need to make this change in `kube/app/ingress-external.yml`

Reply:
Good one. I didn't notice this one. Thanks Jamie. I have updated now.